definitions/checks/check_hotfix_installed.rb

```diff
@@ -46,7 +46,7 @@ def modified_files(package)
   def installed_packages
     packages = []
     repoquery_cmd = execute!('which repoquery')
```

Comment:
Why are we using `which repoquery` here? I don't see any benefit over calling `repoquery` directly, without any path?

Reply:
Yep, does not make sense. Instead maybe we should check if repoquery cmd exist? Which I believe should be there in minimal installation as it comes from yum/dnf-utils

Reply:
the foreman-maintain RPM depends on yum-utils, so it's guaranteed to be there